src/app/c++/ZoneTransit.cxx

```diff
@@ -38,7 +43,7 @@ BOOST_PYTHON_MODULE(libZoneTransit)
     .add_property("content", &ZoneTransit::getContenu);
 
   //Definit la classe Modification (non instantiable, abstraite) et le type shared_ptr<Modification>
```

Comment:
Change the comment: not `shared_ptr<Modification>` anymore, it's `ModificationPtr`